src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/GovUk.Education.ExploreEducationStatistics.Public.Data.Model.csproj

```diff
@@ -15,8 +15,8 @@
 
     <ItemGroup>
         <PackageReference Include="Dapper" Version="2.1.66" />
-        <PackageReference Include="DuckDB.NET.Data.Full" Version="1.3.2" />
```

Comment:
#5835 made the following changes to `DuckDB.NET.Data.Full`.

In **Common.DuckDb**

1.2.1 -> 1.3.2

In **Public.Data.Model**

1.0.2 -> 1.3.2.

Just checking we're covering all of the downgrades that we need to make to `DuckDB.NET.Data.Full` here because this is only one of them.

Also do we know if `InterpolatedSql.Dapper` part of the problem? Just wondering if it's necessary to downgrade it?